src/adapt.js

```diff
@@ -36,17 +36,17 @@ export default function adapt (W1, W2, XYZ, options = {}) {
 	if (!env.M) {
 		if (env.W1 === WHITES.D65 && env.W2 === WHITES.D50) {
 			env.M = [
-				[  1.0479298208405488,    0.022946793341019088,  -0.05019222954313557 ],
-				[  0.029627815688159344,  0.990434484573249,     -0.01707382502938514 ],
-				[ -0.009243058152591178,  0.015055144896577895,   0.7518742899580008  ]
+				[ 1.0479297925449969, 0.022946870601609652, -0.05019226628920524 ],
+				[ 0.02962780877005599, 0.9904344267538799, -0.017073799063418826 ],
+				[ -0.009243040646204504, 0.015055191490298152, 0.7518742814281371 ]
 			];
 		}
 		else if (env.W1 === WHITES.D50 && env.W2 === WHITES.D65) {
 
 			env.M = [
-				[  0.9554734527042182,   -0.023098536874261423,  0.0632593086610217   ],
-				[ -0.028369706963208136,  1.0099954580058226,    0.021041398966943008 ],
-				[  0.012314001688319899, -0.020507696433477912,  1.3303659366080753   ]
+				[ 0.947386632323667, 0.28196156725620036, -0.1708280666484637 ],
+				[ -0.7357288996314816, 1.6804471734451398, 0.035992069603406264 ],
+				[ 0.029218329379919382, -0.05145129980782719, 0.7733468362356041 ]
```

Comment:
These are very different values.

The other changes seem to be the same value at higher precision, but this one is completely different.

Is this correct?

When we use these values in `postcss-preset-env` it dramatically changes the color results.

Reply:
@romainmenke great catch! Accidentally copied in the adapted M cone response as the inverse. I have a fix here: https://github.com/LeaVerou/color.js/pull/360. This one copies the correct matrix, directly calculated by color.js when a D50 <-> D65 conversion occurs.